src/cbmc/bmc.cpp

```diff
@@ -428,6 +429,12 @@ safety_checkert::resultt bmct::run(
   symex.set_message_handler(get_message_handler());
   symex.options=options;
 
+  {
+    const symbolt *init_symbol;
+    if(!ns.lookup(CPROVER_PREFIX "initialize", init_symbol))
+      symex.mode=init_symbol->mode;
+  }  
+
```

Comment:
Agreed - there are cases where value sets might be used without __CPROVER_initialize being present.

It may make sense to briefly remove the default value for _language_mode, do a make -k, and list all the invocations that fail to give everyone a full picture of where else this would possibly be relevant.